packages/mui-material/src/styles/components.d.ts

```diff
@@ -480,6 +480,11 @@ export interface Components<Theme = unknown> {
     styleOverrides?: ComponentsOverrides<Theme>['MuiSvgIcon'];
     variants?: ComponentsVariants['MuiSvgIcon'];
   };
+  MuiSwipeableDrawer?: {
+    defaultProps?: ComponentsProps['MuiSwipeableDrawer'];
+    styleOverrides?: ComponentsOverrides<Theme>['MuiSwipeableDrawer'];
+    variants?: ComponentsVariants['MuiSwipeableDrawer'];
```

Comment:
```suggestion
    defaultProps?: ComponentsProps['MuiSwipeableDrawer'];
```

This is sufficient for fixing the problem that the developer had. Everything else can be dropped.